### PR TITLE
Added support to rename projects on manager

### LINKED ISSUE #10740: Add renaming on project select screen
It would be nice to have a button on the project select screen to be able to rename projects.

### PR DESCRIPTION
**Pull request by nabor**
Closes #10740

### CODE REVIEW

**Comment by saltares** [comment]
You probably want to close either #10962 or this PR. Don't they do the same thing?

**Comment by hpvb** [comment]
Just building and doing a quick test. The code looks good to me. If it works as advertised I'll merge it.

File: editor/project_manager.cpp [hunk 1]
```diff
@@ -883,6 +966,11 @@ void ProjectManager::_load_recent_projects() {
 	tabs->set_current_tab(0);
 }
 
+void ProjectManager::_on_project_renamed() {
+	print_line("PRoject renamed");
```
Comment (hpvb) [inline editor/project_manager.cpp]: Extra capital letter here.

**Review by hpvb — CHANGES_REQUESTED** [review]

File: editor/project_manager.cpp (hunk 1 above)
Comment (nabor) [reply editor/project_manager.cpp]: Yeah and that should not been uploaded. I'm fixing it right away

**Review by hpvb — APPROVED** [review]

**Pull request merged**